ipa-core/src/net/server/mod.rs

```diff
@@ -295,18 +280,20 @@ async fn rustls_config(
     for cert in network
         .peers()
         .iter()
-        .filter_map(|peer| peer.certificate.as_ref())
+        .filter_map(|peer| peer.certificate.clone())
     {
         // Note that this uses `webpki::TrustAnchor::try_from_cert_der`, which *does not* validate
         // the certificate. That is not required for security, but might be desirable to flag
         // configuration errors.
         trusted_certs.add(cert)?;
     }
-    let verifier = AllowAnyAnonymousOrAuthenticatedClient::new(trusted_certs);
+    let client_verifier = WebPkiClientVerifier::builder(trusted_certs.into())
+        .allow_unauthenticated()
+        .build()
+        .unwrap();
```

Comment:
`expect`?

Reply:
Ok

Reply:
@andyleiserson may chime in too, my understanding is that we don't authenticate MPC clients, only MPC-to-MPC channels are authenticated using client-cert auth. If MPC helper opts-out from cert-based authentication, the client identity won't be set and request handlers (like [this](https://github.com/private-attribution/ipa/blob/fa79ceefc637b58a52d66b78c667ecb496d7d909/ipa-core/src/net/server/handlers/query/prepare.rs#L15)) will fail the request